web/src/Routes.tsx

```diff
@@ -28,7 +30,7 @@ const Routes: React.FC = () => (
   <Suspense fallback={<Loading />}>
     <Switch>
       <Route exact path="/design-samples" component={DesignSamplesPage} />
-      <Route path="/merchant/" component={MerchantRoutes} />
+      <Route path={MERCHANT_ROOT} component={MerchantRoutes} />
```

Comment:
Some routes have literal paths; some routes have paths imported from constants. What's the rationale behind this distinction? Can we have some consistency in this routes file?

Reply:
@xlai20 
Because this commit https://github.com/googleinterns/gpay-group-buy/pull/158/commits/e31924f4c47d623aef0fe8925717002fb601ec47 adds `MERCHANT_ROOT` for `ListingCollection` so that we don't have to directly pass in `/merchant/` to `listingRootPath` props, as I did not really want components don't have to be aware of what merchant root is. 
And since I added `MERCHANT_ROOT` to constants, I felt that the `Routes.tsx` should use it too to be consistent. (I didn't want the case where someone changes the path in `Route` and make `MERCHANT_ROOT` outdated.
At the same time I am reluctant to change the rest of the routes to be imported because they will only be used once in this file and mainly because it would be less clear how paths are routed in the `Routes.tsx` file.
For example, if we have a constant `const CUSTOMER_ROOT = '/'`, and we do
```
      <Route path={MERCHANT_ROOT} component={MerchantRoutes} />
      <Route path={CUSTOMER_ROOT} component={CustomerRoutes} />
```
Then it becomes unclear if path '/' is used or not (which is kind of important in routing), and also, it looks like the order can be changed up like so:
```
      <Route path={CUSTOMER_ROOT} component={CustomerRoutes} />
      <Route path={MERCHANT_ROOT} component={MerchantRoutes} />
```
Which should not be allowed because `path='/'` should be the last path, else every single route will be routed to `CustomerRoutes`.
Hmm what do you think I should do to make this better?